app/views/RoomsListView/ListHeader/index.js

```diff
@@ -1,28 +1,80 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import Sort from './Sort';
-import Encryption from './Encryption';
+import { withTheme } from '../../../theme';
+import I18n from '../../../i18n';
+import { Item } from '../../../containers/List';
```

Comment:
Check how this import is done for `List` on other places of the app.
The idea here is to always call `List.Item`, so we follow the same pattern.